portfolio/src/main/webapp/index.html

```diff
@@ -9,10 +9,101 @@
   <body>
     <div id="content">
```

Comment:
should the content div be wrapped around the whole thing so your entire page can have the same alignment?